test/etl_test.py

```diff
@@ -131,3 +129,61 @@ def test_ferc1_solo_etl(datastore_fixture,
         pudl_settings_fixture,
         datapkg_bundle_name='ferc1-solo',
         clobber=True)
+
+
+class TestFerc1Datastore:
+    """Validate the Ferc1Datastore functions."""
+
+    ds = pudl.extract.ferc1.Ferc1Datastore(sandbox=True)
+
+    def test_ferc_folder(self):
+        """Spot check we get correct folder names per dataset year."""
+        assert ds.ferc_folder(1994) == "FORMSADMIN/FORM1/working"
+        assert ds.ferc_folder(2001) == "UPLOADERS/FORM1/working"
+        assert ds.ferc_folder(2002) == "FORMSADMIN/FORM1/working"
+        assert ds.ferc_folder(2010) == "UPLOADERS/FORM1/working"
+        assert ds.ferc_folder(2015) == "UPLOADERS/FORM1/working"
+
+
+class TestExcelExtractor:
+    """Verify that we can lead excel files as provided via the datastore."""
+
+    def test_excel_filename_eia860(self):
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Method could be a function](https://app.codacy.com/manual/zaneselvans/pudl/pullRequest?prid=5437498)